docs/pipeline.md

```diff
@@ -127,6 +127,8 @@ container.write(Context1, [11])
 container.read(Context0) // 1
 container.read(Context1) // [11]
 
+const pipeline = createPipeline<number, number>()
```

Comment:
Although this solves a compile-time unresolved variable problem, this example still runs with an error, since there is no pipeline.use(...) defined for it.
Should here be a usage example of accessing pipeline's internal container?

Reply:
Sounds good, we are planning to improve the context API and even consider `farrow-context` to be standalone package